Gemfile

```diff
@@ -45,7 +45,7 @@ gem "octokit", "~> 4.0"
 gem 'docker-api', :require => 'docker'
 gem 'uuid'
 gem 'listen', '3.0.8'
-
+gem 'akamai-edgegrid', '~> 1.0', '>= 1.0.6'
```

Comment:
Do you want to lock this down to an actual version?

Reply:
Actually wondering if we can trim a bunch of stuff here since we don't use awestruct anymore.

Reply:
👍